packages/cli/src/ui/components/SettingsDialog.tsx

```diff
@@ -38,6 +38,8 @@ interface SettingsDialogProps {
   settings: LoadedSettings;
   onSelect: (settingName: string | undefined, scope: SettingScope) => void;
   onRestartRequest?: () => void;
+  availableTerminalHeight?: number;
```

Comment:
nit: make availableTerminalHeight and availableTerminalWidth required rather than optional. We should always know this and will avoid bugs if we accidentally stopped passing them in as could happen if someone had merge conflicts applying your changes.